basesandbox/src/execution/mod.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright 2020 Kodebox, Inc.
```

Comment:
Though current Foundry codebase is largely following pre-2018 edition convention, it seems better to shy away from the use of `mod.rs`. In this case, move `mod.rs` up to the parent directory and rename it to `execution.rs`.

https://doc.rust-lang.org/edition-guide/rust-2018/module-system/path-clarity.html#no-more-modrs

Reply:
Wow. I really didn't know that

Reply:
@majecty What do you think of this? Should I apply this rule for my work? (Since my work is not related to codechain codebase, so might be ok to be try the latest convention?)

Reply:
It seems good. Since many other modules in Foundry are using `mod.rs`, how about sharing this with all other team members?